webapp/channels/src/components/login/login.tsx

```diff
@@ -363,8 +363,6 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
     const onWindowFocus = useCallback(() => {
         if (extraParam === Constants.SIGNIN_VERIFIED && emailParam) {
             passwordInput.current?.focus();
-        } else {
-            loginIdInput.current?.focus();
```

Comment:
This makes the header logo and `Don't have an account?` being skipped on tab. Force focus to login input field is already handled by `autoFocus={true}` at https://github.com/mattermost/mattermost/blob/e7c6d08a7eeee811a577b10b8b73f21b105af1ad/webapp/channels/src/components/login/login.tsx#L823